### PR TITLE
♻️ Maintenance/Director-v2 to use settings library

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

Migrates the director-v2 to use the settings-library

### CODE REVIEW

File: services/director-v2/requirements/_base.in [hunk 1]
```diff
@@ -4,6 +4,10 @@
 # NOTE: ALL version constraints MUST be commented
 --constraint ../../../requirements/constraints.txt
 
+-r ../../../packages/models-library/requirements/_base.in
```
Comment (pcrespov) [inline services/director-v2/requirements/_base.in:7]: use long version of the flags ``--requirement``

File: services/director-v2/src/simcore_service_director_v2/modules/dask.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# def setup(app: FastAPI, settings: DaskConfig) -> None:
+#     def on_startup() -> None:
```
Comment (pcrespov) [inline services/director-v2/src/simcore_service_director_v2/modules/dask.py:2]: is this file needed?

**Review by pcrespov — APPROVED** [review]
🎉  thx for helping upgrading settings
some minor

File: services/director-v2/tests/unit/test_core_settings.py [hunk 3]
```diff
@@ -36,6 +36,7 @@ def test_create_registry_settings(project_env_devel_environment, monkeypatch):
     assert settings.api_url == "https://registry:5000/v2"
 
 
+@pytest.mark.skip(reason="registry will not be used from director-v2")
```
Comment (GitHK) [inline services/director-v2/tests/unit/test_core_settings.py:39]: well maybe just remove this test?

**Review by GitHK — APPROVED** [review]
Nice, I so not look forward to merge this 🤣

File: services/director-v2/tests/unit/test_core_settings.py (hunk 3 above)
Comment (sanderegg) [reply services/director-v2/tests/unit/test_core_settings.py:39]: well I think you are removing it. I don't want to create more merge conflicts than needed.

File: services/director-v2/src/simcore_service_director_v2/modules/dask.py (hunk 2 above)
Comment (sanderegg) [reply services/director-v2/src/simcore_service_director_v2/modules/dask.py:2]: no. good one.

File: services/director-v2/tests/unit/test_core_settings.py (hunk 3 above)
Comment (sanderegg) [reply services/director-v2/tests/unit/test_core_settings.py:39]: ok removing it. removal is easier in conflicts

Comment (sanderegg) [reply services/director-v2/tests/unit/test_core_settings.py:39]: ok removing it. removal is easier in conflicts
